app/forms/provider_interface/interview_wizard.rb

```diff
@@ -104,6 +104,21 @@ def clear_state!
       @state_store.delete
     end
 
+    def self.from_model(store, interview, step = 'input')
+      wizard = new(store, { current_step: step })
```

Comment:
I was wondering if we could simplify this somehow so that we don't need to worry if we make any additions to the model, but I don't think there is any straightforward way to do it unless we switch `attr_accessor` to `attributes` and then grab `attribute_names`. 

Maybe something to keep in mind and do as part of the wizard work.